localstack/services/events/scheduler.py

```diff
@@ -8,6 +9,41 @@
 
 LOG = logging.getLogger(__name__)
 
+CRON_REGEX = re.compile(r"\s*cron\s*\(([^\)]*)\)\s*")
+RATE_REGEX = re.compile(r"\s*rate\s*\(([^\)]*)\)\s*")
+
+
+def convert_schedule_to_cron(schedule):
```

Comment:
sidenote:💡 That method could be candidate for an `_aws` [internal endpoint](https://docs.localstack.cloud/references/internal-endpoints/) similar to https://github.com/localstack/localstack/pull/10710

It's probably not worth here because we can get good parity test coverage using `tests_put_rule_with_schedule_cron`.